spec/migrations/20150817213409_clear_tenant_seed_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+require Rails.root.join("db/migrate/20150817213409_clear_tenant_seed")
+
+describe ClearTenantSeed do
+  class ClearTenantSeed::Tenant < ActiveRecord::Base
+  end
+  let(:tenant_stub) { migration_stub(:Tenant) }
+
+  migration_context :up do
+    it "works with no tenants" do
+      expect { migrate }.not_to raise_error
+    end
+
+    it "works with one tenants" do
+      tenant = tenant_stub.create(:name => 'My Company')
+      expect { migrate }.not_to raise_error
```

Comment:
This is sort of a pointless test...because it should never error...just call migrate and if it blows up it fails the test anyway.